core/state/statedb.go

```diff
@@ -211,7 +212,12 @@ func (self *StateDB) UpdateStateObject(stateObject *StateObject) {
 		self.db.Put(stateObject.CodeHash(), stateObject.code)
 	}
 	addr := stateObject.Address()
-	self.trie.Update(addr[:], stateObject.RlpEncode())
+	data, err := rlp.EncodeToBytes(stateObject)
+	if err != nil {
+		glog.Errorf("can't encode object at %x: %v", addr[:], err)
+		return
```

Comment:
Change this so that indeed an error is either returned (and checked and panicked) or panic here.